packages/graphback-codegen-schema/src/plugin/SchemaCRUDPlugin.ts

```diff
@@ -0,0 +1,234 @@
+import { GraphbackPlugin, GraphbackCoreMetadata, ModelDefinition } from '@graphback/core'
+import { mergeSchemas } from "@graphql-toolkit/schema-merging"
+import { GraphQLID, GraphQLInputObjectType, GraphQLList, GraphQLNonNull, GraphQLObjectType, GraphQLSchema } from 'graphql';
+import * as pluralize from "pluralize";
+import { gqlSchemaFormatter, jsSchemaFormatter, tsSchemaFormatter } from '../writer/schemaFormatters';
+import { printSortedSchema } from '../writer/schemaPrinter';
+
+/**
+ * Configuration for Schema generator CRUD plugin
+ */
+export interface SchemaCRUDPluginConfig {
+    // output format for schema string
+    format: 'ts' | 'js' | 'gql'
```

Comment:
This config does not seem appropriate for the `SchemaCRUDPluginConfig`. To me this plugin should logically contain the following:

```
// Generate create method
    create?: boolean;
    // Generate update method
    update?: boolean;
    //tslint:disable-next-line
    delete?: boolean;
    // Generate find method
    find?: boolean;
    // Generate findAll method
    findAll?: boolean;
    // Generate subscription for create operation
    subCreate?: boolean;
    // Generate subscription for update operation
    subUpdate?: boolean;
    // Generate subscription for delete operation
    subDelete?: boolean;
    // Disable codegen
    disableGen?: boolean;
```

And we could have another plugin "SchemaFormatterPlugin" which is specifically for formatting the schema as ts, js, gql etc.

I won't change anything before I hear your thoughts on this.